dashboard/config/environments/adhoc.rb

```diff
@@ -36,12 +36,6 @@
   # the I18n.default_locale when a translation can not be found).
   config.i18n.fallbacks = true
 
-  # Whether or not to display pretty apps (formerly called blockly).
-  config.pretty_apps = true
-
-  # Whether or not to display pretty shared js assets
-  config.pretty_sharedjs = true
-
```

Comment:
Brent commented on this as well -- see my response there. I'll revert the changes to adhoc.rb.